src/core/mod.rs

```diff
@@ -452,7 +460,7 @@ impl Tox {
     pub fn wait(&self) {
         unsafe {
             let delay = ll::tox_iteration_interval(self.raw);
-            sleep_ms(delay);
+            sleep(Duration::new(0, delay));
```

Comment:
This is my fault..
I use `rust-night`, `sleep_ms` there is a deprecated warning.